contracts/bridge-wrappers/Synthetix_Optimism_BridgeWrapper.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
```

Comment:
nit: Thoughts on naming this `SnxOptimismBridgeWrapper`

Reply:
Sounds good. I don't quite like the snakecase names anyway.